### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gpeuc**
Added links to design file & guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,3 +110,8 @@ Users are currently unable to filter the Guided Answers on Products or Component
 ## Known Issues
 
 https://github.com/SAP/guided-answers-extension/issues?q=is%3Aopen+is%3Aissue+label%3Abug
+
+## Design Sources
+
+Figma file (Read Only) - https://www.figma.com/file/XSZw3w81MYY0QPe1BoUcBO/Guided-Answers?node-id=1%3A4211&t=vVdqU90dr8pTwJQP-1 
+General Design Guidelines - https://wiki.one.int.sap/wiki/display/fioritools/SAP+Fiori+tools+Home
```
Comment (Klaus-Keller) [inline README.md:117]: @gpeuc, you'll need a logon to access this link. Sure we should publish this here?

File: .changeset/short-bikes-nail.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (Klaus-Keller) [inline .changeset/short-bikes-nail.md:1]: change sets are required to produce new versions of the extension. Please remove the change set, as changes to readme/documentation should not result in new extension versions.

**Review by Klaus-Keller — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gpeuc) [reply README.md:117]: @Klaus-Keller Figma link is public read-only. No login necessary. You might be right about Wiki, it seems it requires SAP SSO. Can you remove it then please?

Comment (Klaus-Keller) [reply README.md:117]: @gpeuc, yes, I was talking about the wiki link only. I'll see if I can do changes, not sure as it is your fork you created this PR from.

**Review by Klaus-Keller — APPROVED** [review]
Seems I'm able to do the changes.

**Pull request merged**